lib/salve/errors.js

```diff
@@ -136,9 +134,9 @@ inherit(ElementNameError, SingleNameError);
  * @constructor
  * @extends module:patterns~ValidationError
  * @param {Array.<module:name_patterns~Base>} names_a The name of the first
- * XML entities at stake.
+ * XML entity at stake.
```

Comment:
Actually, the plural is correct here. What has to be corrected is "The name" => "The names" So:

> The names of the first XML entities at stake.